client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -203,7 +203,7 @@ class PlanningListSubNavComponent extends React.Component<IProps, IState> {
                         </div>
                     )}
 
-                    <ButtonGroup align="end">
+                    <ButtonGroup className = "btnGroup" align="end">
```

Comment:
not sure why this `ButtonGroup` has `btnGroup` class but the other one not, it should be either more specific or set on all of them

Reply:
we're applying that `btnGroup` because the modifications are only needed here. If we alter the `ButtonGroup`, the changes may be reflected elsewhere.

Reply:
I get this, but then we should use some more specific class name here instead of `btnGroup` which would make sense to be on all of those and not just here